app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+class RentsController < ApiController
+  before_action :authenticate_user!
+
+  include Wor::Paginate
+
+  def create
+    @rent = Rent.new(rent_create_params)
+    if !@rent.valid?
+      render json: { errors: @rent.errors }
```

Comment:
how is the format of this json? As we talk the other day, it's a good practice having a unique format of responding errors. Having this code here, will make you have repeated login cross controllers. Let's move somewhere else where we can reuse it